### ISSUE TITLE
[FE] Update Broken Accessibility Tests for Review Page

### ISSUE THREAD

**Issue by mdewey** [department-of-veterans-affairs/va.gov-team#14175]
## Description
The platform team has updated the built-in [`FormTester` tools](https://github.com/department-of-veterans-affairs/vets-website/blob/healthcare-12282-landing-page/src/platform/testing/e2e/cypress/support/form-tester/index.js#L27) to now include a11y checks. This update has caused two of our current e2e test to start to fail. 

## Tasks
- [ ] Update the review page to not have the accessibility issues
- [ ] ensure that tests pass

## Acceptance Criteria
- [ ] All e2e test passing.

**Comment by julieadair1** [comment]
@mdewey what needs to be done in order to complete this epic?
cc: @jparker0425

**Comment by jparker0425** [comment]
Looks like code was merged and items were closed a while back @julieadair1 . Closing this out.
cc: @mdewey if additional insight is needed
